doc/releasenotes/7_0_3_release_notes.md

```diff
@@ -0,0 +1,11 @@
+This update fixes several regressions that were deemed significant enough to be backported to the release branch. 
+
+## Bugs Fixed
+
+* healthcheck : Should receive healthcheck updates from all tablets in cells_to_watch #6857
+* vtgate : Fix error around breaking of multistatements #6824
+* backport : checkNoDB should not require tables to be present #6788
```

Comment:
Let's update the categories for these fixes:
1. healthcheck -> Vtgate
2. vtgate -> Vtgate
3. backport -> Vttablet
4. backport -> Build

Reply:
Done